scripts/install_full.bat

```diff
@@ -11,5 +11,4 @@ set cmd="python .\scripts\get-python-version.py"
 for /f "tokens=*" %%v in (' %cmd% ') do set SHORT_PY_VER=%%v
 pip install -r .\src\azure-cli\requirements.%SHORT_PY_VER%.Windows.txt
 
-echo %OUTPUT%
```

Comment:
what is the value of %OUTPUT%? Why remove it?

Reply:
%OUTPUT% is not a variable in those *.bat scripts context,  Azure DevOps and Windows Batch sctipt. The functionality is going to ECHO a non-existing variable. Can tell from [line 360 of build 20190702.7](https://dev.azure.com/azure-sdk/public/_build/results?buildId=60891&view=logs&j=796343fd-f04d-59ce-7e73-c4eab21e4249&t=3e5b4737-4f47-5027-dc25-be482d8c1eaf&l=360) . 

So, I think it's trivial.